apps/system/js/rocketbar.js

```diff
@@ -257,8 +257,10 @@
         case 'appforeground':
         case 'appopened':
         case 'utility-tray-overlayopening':
-          this.hideResults();
-          this.deactivate();
+          eventSafety(window, 'iac-search-results', function() {
```

Comment:
Won't this cause a 100ms delay in the appforeground/appopened case? I think we should avoid the delay where possible.

What if we instead just check the state of the utility tray when we get an iac-search-result message? We could listen to the utility-tray-overlayopened/utility-tray-overlayclosed events and keep the value in a local variable. What do you think?